demos/tests/cases.py

```diff
@@ -107,7 +107,8 @@ def single_option_cases(key, *args):
             ModelArg('repvgg-a0'),
             ModelArg('repvgg-b1'),
             ModelArg('repvgg-b3'),
-            ModelArg('resnet-50-caffe2')),
+            ModelArg('resnet-50-caffe2'),
+            ModelArg('shufflenet-v2-x0.5')),
```

Comment:
do we really need this model in test cases?
I believe classification demo support a lot of models and their list decreased for time reducing for testing

Reply:
Test case deleted.

Reply:
I not agree, please return model to test cases. We do test all models supported by demo. Having model listed in models.lst file also trigger download and convert steps, which otherwise will not be tested